docs/about-lisk/bridged-tokens.md

```diff
@@ -22,10 +22,10 @@ If you want to add a token to the list, please check out the guide [Bridging an
 
 ## Lisk Mainnet
 
-| Bridged Tokens Mainnet | Symbol | L1 Token          | L2 Token           |
-| :----------------- | :----- |:----------------- | :----------------- |
+| Bridged Tokens Mainnet | Symbol | L1 Token          | L2 Token           | Bridges |
+| :--------------------- | :----- |:----------------- | :----------------- | :----- |
 
 ## Lisk Sepolia
 
-| Bridged Tokens Sepolia | Symbol | L1 Token          | L2 Token           |
-| :----------------- | :----- |:----------------- | :----------------- |
+| Bridged Tokens Sepolia | Symbol | L1 Token          | L2 Token           | Bridges |
+| :--------------------- | :----- |:----------------- | :----------------- | :----- |
```

Comment:
The `Bridges` column is empty for Lisk Sepolia. I would either remove this column or add `N/A` as a value to each row

Reply:
@Tschakki, please also remove the `Bridges` column from the English version as well.

Reply:
done 👍